### DISCUSSION TITLE
Matching - Audible vs the others

### DISCUSSION THREAD

**Discussion by BDaddyG** [advplyr/audiobookshelf#619] [Q&A] [answered]
I have a question about Matching to Audible.

I like to name my book folders with the date the book was published, ie: "Ash (2018)"

For example, the path for this book would then be: Tom Abrahams/The Alt Apocalypse/01 - Ash (2018)/file(s) for book

When doing a match, "Ash (2018)" is listed as the search title, and "Tom Abrahams" is listed in the Author filed.

With Google Books and Open Library, performing the search will return a result, but nothing is returned for Audible.

If I remove (2018) from the search title, then the Audible search will return a result.

Is this a restriction placed by the Audible API or a restriction of AudioBookShelf?

Thanks!

**Comment by advplyr** [comment, marked as answer]
It must be with Audible unless there is a better way to use the Audible API. Abs is passing in the title you are sending.
That is happening here: https://github.com/advplyr/audiobookshelf/blob/master/server/providers/Audible.js#L67

**Reply by BDaddyG** [reply]
Thanks for the response.
